templates/docs/starter/{{ cookiecutter.__folder_name }}/Makefile

```
@@ -95,6 +95,12 @@ rtd-prepare:  ## Prepare environment on Read the Docs
 .PHONY: rtd-pr-preview  ## Build pull request previews on Read the Docs
 rtd-pr-preview: rtd-prepare .venv/bin/python ## Build pull request preview on Read the Docs
 	$(SPHINXBUILD) -b html $(ALLSPHINXOPTS) ${READTHEDOCS_OUTPUT}/html/
+
+
+
+.PHONY: build
```

Remove extra blank lines.

```suggestion
.PHONY: build
```